src/mono/wasm/debugger/tests/debugger-test/debugger-evaluate-test.cs

```diff
@@ -1442,6 +1461,36 @@ public static void Run()
     }
 }
 
+public static class NestedWithSameNames
```

Comment:
I added some members:
```csharp
public static class NestedWithSameNames
{
    public static int StaticField1 = 30;
    public static string StaticProperty1 => "StaticProperty3";
    public static string StaticPropertyWithError => throw new Exception("not implemented V3");

    public static class B
    {
        public static int StaticField1 = 60;
        public static string StaticProperty1 => "StaticProperty6";
        public static string StaticPropertyWithError => throw new Exception("not implemented V6");

        public static class NestedWithSameNames
        {
            public static class B
            {
                public static int NestedWithSameNames = 90;
                public static int StaticField1 = 40;
                public static string StaticProperty1 => "StaticProperty4";
                public static string StaticPropertyWithError => throw new Exception("not implemented V4");

                public static void Run()
                {
                    var a = 0;
                }
            }
        }

        public static class NestedWithDifferentName
        {
            public static class B
            {
                //public static int NestedWithSameNames = 90;
                public static int StaticField1 = 70;
                public static string StaticProperty1 => "StaticProperty7";
                public static string StaticPropertyWithError => throw new Exception("not implemented V7");

                public static void Run()
                {
                    var a = 0;
                }
            }
        }
    }

    public static void Evaluate()
    {
        B.NestedWithSameNames.B.Run();
    }
}
```

.. which breaks one of the tests. Also, you can add a few more like:
`NestedWithSameNames.B.NestedWithDifferentName.B.StaticProperty1`, `NestedWithSameNames.B.NestedWithSameNames.B.StaticField1`, `B.NestedWithDifferentName.B.StaticProperty1`.